### PR TITLE
Conditional tic.R templates

### LINKED ISSUE #235: Default tic.R still has check for ci_on_travis()
Should be adapted to the deployment platform selected in `use_tic()`.

### PR DESCRIPTION
**Pull request by pat-s**
fixes #235 

Conditioned on what has been set for `deploy` during `use_tic()`.

By default the template without deployment is used if called standalone with `repo_type` requiring always input from the user.

(`use_tic_r()` is exported since today.)

# Example

```r
use_tic_r("package", deploy_on = "ghactions")
```

results in 

```r
# installs dependencies, runs R CMD check, runs covr::codecov()
do_package_checks()

if (ci_on_ghactions()) {
  # creates pkgdown site and pushes to gh-pages branch
  do_pkgdown()
}
```

### CODE REVIEW

**Comment by pat-s** [comment]
Merging because I need it for the ropensci blogpost.

File: R/use_tic.R [hunk 1]
```diff
@@ -307,7 +307,7 @@ use_tic <- function(wizard = interactive(),
 
   rule(left = "tic")
 
-  use_tic_r(repo_type = detect_repo_type())
+  use_tic_r(repo_type = detect_repo_type(), deploy_on = )
```
Comment (krlmlr) [inline R/use_tic.R]: Is this intended?

Comment (pat-s) [reply R/use_tic.R]: Thanks, wondering how that even passed CI. Let's default to "none".

Comment (pat-s) [reply R/use_tic.R]: Ah its in `use_tic()` which has no tests. We really need some..

**Pull request merged**